src/App.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div id="app">
+    <div id="nav">
+      <span v-if="isLoggedIn"> | <a @click="logout">Logout</a></span>
+    </div>
+    <router-view/>
+  </div>
+</template>
+
+<script>
+  export default {
+    computed : {
+      isLoggedIn : function(){ return this.$store.getters.isLoggedIn}
+    },
+    methods: {
+      logout: function () {
+        this.$store.dispatch('auth/logout')
+        .then(() => {
+          this.$router.push('/login')
```

Comment:
This means after I logout you redirect me to login? If yes, I guess it's not best UX, if say I want to browse public doctors list unauthenticated.